Flaky `platform/fabric/core/generic/msp/service_test.go`

### What happened?

```bash
--- FAIL: TestService_IsMe (0.03s)
    --- FAIL: TestService_IsMe/True (0.00s)
        service_test.go:277: 
            	Error Trace:	/home/runner/work/fabric-smart-client/fabric-smart-client/platform/fabric/core/generic/msp/service_test.go:277
            	Error:      	Should be true
            	Test:       	TestService_IsMe/True
FAIL
```

### How can we reproduce it?

Flaky via GH CI

### Environment

- FSC version (e.g., latest main, v0.9.0): main
- Go Version (e.g., go1.25.1 darwin/arm64): see go.mod
- Operating system (e.g., Linux, MacOS): GH CI


### Additional Information

_No response_

--- a/platform/fabric/core/generic/msp/service_test.go
+++ b/platform/fabric/core/generic/msp/service_test.go
@@ -265,6 +265,7 @@ func TestService_SetDefaultIdentity(t *testing.T) {
 		require.Equal(t, initialSid, mspService.DefaultSigningIdentity())
 	})
 }
+
 func TestService_IsMe(t *testing.T) {
 	t.Parallel()
 	id := view.Identity("id1")

--- a/platform/fabric/core/generic/msp/service_test.go
+++ b/platform/fabric/core/generic/msp/service_test.go
@@ -265,22 +265,22 @@ func TestService_SetDefaultIdentity(t *testing.T) {
 		require.Equal(t, initialSid, mspService.DefaultSigningIdentity())
 	})
 }
-
 func TestService_IsMe(t *testing.T) {
 	t.Parallel()
-	mspService, signerService, _, _ := setup(t)
 	id := view.Identity("id1")
 
 	t.Run("True", func(t *testing.T) {
 		t.Parallel()
+		mspService, signerService, _, _ := setup(t)
 		signerService.IsMeReturns(true)
-		require.True(t, mspService.IsMe(context.Background(), id))
+		require.True(t, mspService.IsMe(t.Context(), id))
 	})
 
 	t.Run("False", func(t *testing.T) {
 		t.Parallel()
+		mspService, signerService, _, _ := setup(t)
 		signerService.IsMeReturns(false)
-		require.False(t, mspService.IsMe(context.Background(), id))
+		require.False(t, mspService.IsMe(t.Context(), id))
 	})
 }
 

